Sources/PointFree/Account/Change.swift

```diff
@@ -0,0 +1,392 @@
+import Css
+import Either
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Optics
+import Prelude
+import Styleguide
+import Tuple
+
+// MARK: Middleware
+
+let subscriptionChangeShowResponse =
+  filterMap(require1 >>> pure, or: loginAndRedirect)
```

Comment:
we do this a lot so maybe we just finally make a `requireCurrentUser` middleware transformers to act on `T2<User?, Z>`